src/modules/ecommerce/components/ProviderProfileSection.vue

```diff
@@ -0,0 +1,409 @@
+          >
+            <span
+              v-if="showEmailValue"
+              class="flex-item--grow"
+            >
+              <span v-if="selectedUser.email">
+                {{ selectedUser.email }}
+              </span>
+              <AppContent
+                v-else
+                entry="form"
+                field="error.missing_email"
+              />
+            </span>
+            <AppContent
+              v-else
+              entry="user"
+              field="email_address_label"
+            />
+            <QIcon
+              size="1.5rem"
+              :color="selectedUser.emailVerified ? 'positive' : 'negative'"
+              :name="selectedUser.emailVerified ? icons.matCheckCircle : icons.matAnnouncement"
+            >
+              <AppContent
+                v-if="canValidateEmail && !selectedUser.emailVerified"
+                tag="QTooltip"
+                entry="prompt"
+                field="validate"
+              />
+            </QIcon>
+          </div>
+        </div>
+
+        <div
+          v-show="isCurrentUser || selectedUser.locationName"
+          class="row q-mt-sm"
+        >
+          <AppSwitchableEditor
+            :value="locationValue"
+            :active="isCurrentUser && selectedUserLocations.length <= maxNbLocations"
+            :custom-save="updateUserFn('locations')"
+            :allow-falsy-save="true"
+            tag="div"
+            class="col"
+          >
+            <template v-slot:default>
+              <div class="row items-center">
+                <div class="ellipsis">
+                  {{ selectedUser.locationName }}
+                </div>
+              </div>
+            </template>
+            <template v-slot:placeholder>
+              <AppContent
+                class="switchable-editor-placeholder"
+                entry="places"
+                field="address_placeholder"
+              />
+            </template>
+            <template v-slot:edition="{ content, saveDraft }">
+              <PlacesAutocomplete
+                :initial-query="locationValue"
+                :label="$t({ id: 'places.address_placeholder' })"
+                @selectPlace="loc => prepareUpdatedLocations(loc, saveDraft)"
+              />
+            </template>
+          </AppSwitchableEditor>
+        </div>
+
+        <div
+          v-show="isCurrentUser || selectedUser.description"
+          class="row q-mt-md"
+        >
+          <AppSwitchableEditor
+            tag="p"
+            class="text-justify"
+            :value="selectedUser.description"
+            :active="isCurrentUser"
+            :custom-save="updateUserFn('description')"
+            :input-label="$t({ id: 'user.account.my_description_label' })"
+            allow-falsy-save
+            input-type="textarea"
+          />
+        </div>
+
+        <div
+          v-if="showDeliveryFee && (isCurrentUser || selectedUser.deliveryFee)"
```

Comment:
This seems to ignore the case when delivery is free: explicitly 0, not the same as unspecified even if it is the same result during payment.
When unspecified the owner is likely to ask for an offline payment. `0` value set by owner can reassure the buyer.